packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -1,18 +1,11 @@
-import {
-  MutationTypes,
-  CozyLink,
-  getDoctypeFromOperation
-} from 'cozy-client'
+import { MutationTypes, CozyLink, getDoctypeFromOperation } from 'cozy-client'
```

Comment:
Some may have different prettier/formatting rules. Could you check it's not you?

Reply:
Patrick didn't format its code correctly using prettier, and as I don't have added a git hook yet... So, no it's not @y-lohse fault ;)